utils/src/main/java/com/redhat/lightblue/migrator/utils/DAOFacadeBase.java

```diff
@@ -31,12 +33,40 @@
 
     protected final D legacyDAO, lightblueDAO;
 
+    private EntityIdStore entityIdStore = null;
+
+    public EntityIdStore getEntityIdStore() {
+        return entityIdStore;
+    }
+
+    public void setEntityIdStore(EntityIdStore entityIdStore) {
+        this.entityIdStore = entityIdStore;
+
+        try {
+            Method method = lightblueDAO.getClass().getMethod("setEntityIdStore", EntityIdStore.class);
+            method.invoke(lightblueDAO, entityIdStore);
```

Comment:
I thought about this, but I decided for an approach which does not require any changes in the current model, entities or dao apis. It has it's disadvantages for sure.